pkg/scopes/scopes.go

```diff
@@ -47,14 +48,13 @@ func GetScopedTopics(scope, appID string, metadata map[string]string) []string {
 	return topics
 }
 
-// GetAllowedTopics return the all topics list of params allowedTopics.
-func GetAllowedTopics(metadata map[string]string) []string {
+func getParamTopics(param string, metadata map[string]string) []string {
 	var (
 		existM = map[string]struct{}{}
 		topics = []string{}
 	)
 
-	if val, ok := metadata[AllowedTopics]; ok && val != "" {
+	if val, ok := metadata[param]; ok && val != "" {
```

Comment:
Please, make the match case-insensitive. We are trying to make all metadata properties match case-insensitive.

Reply:
Sounds good, updated in c0718be3